core/sector_storage/impl/remote_worker.cpp

```diff
@@ -3,25 +3,44 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <boost/beast/http.hpp>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+#include "primitives/jwt/jwt.hpp"
+
+#include "common/uri_parser/uri_parser.hpp"
+#include "drand/impl/http.cpp"
+#include "primitives/piece/piece_data.hpp"
 #include "sector_storage/impl/remote_worker.hpp"
 
 namespace fc::sector_storage {
+  using drand::http::ClientSession;
+  using primitives::piece::MetaPieceData;
+  using primitives::piece::ReaderType;
+  namespace beast = boost::beast;
+  namespace net = boost::asio;
+  using tcp = net::ip::tcp;
+  namespace uuids = boost::uuids;
+  namespace http = boost::beast::http;
+  using common::HttpUri;
+  using primitives::jwt::kAdminPermission;
 
   outcome::result<std::shared_ptr<RemoteWorker>>
   RemoteWorker::connectRemoteWorker(io_context &context,
                                     const std::shared_ptr<CommonApi> &api,
                                     const Multiaddress &address) {
-    auto token = "stub";  // get token from common api
-
+    auto token = "stub";
```

Comment:
Is it a correct token?

Reply:
That was before me, I will change it in next PR